prisma/schema.prisma

```diff
@@ -229,3 +231,28 @@ model DepositHead {
 
   @@map("deposit_head")
 }
+
+model MaspTransaction {
+  id               Int                 @id @default(autoincrement())
+  created_at       DateTime            @default(now()) @db.Timestamp(6)
+  updated_at       DateTime            @default(now()) @updatedAt @db.Timestamp(6)
+  transaction_hash String              @db.VarChar
+  block_hash       String              @db.VarChar
+  asset            String              @db.VarChar
```

Comment:
We've been using asset identifier in the MASP prototypes. What do you think about that?

Reply:
SGTM